### PR TITLE
Plotly.js assets trick

### PR DESCRIPTION
**Pull request by nicolaskruchten**
Long-outstanding documentation for these Plotly.js variations

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Looks like docs need the `cryptography` downgrade on Py3.6 https://github.com/plotly/dash/commit/aab619275be65d1284e4f0467a7d9057ce8391e6 (until a new dash is published and included)

File: dash_docs/chapters/external_resources/index.py [hunk 1]
```diff
@@ -353,6 +353,32 @@ def b64_image(image_filename):
...
+        * take advantage of more desirable behaviour of a version of Plotly.js that is **less recent** than the one that is included in the currently-installed version of `dash_core_components` (or Dash Enterprise Design Kit). Note that this situation should be rare and short-lived, as we strive to make more-recent Plotly.js versions totally backwards-compatible with older versions!
+        * use a Plotly.js partial or custom bundle that only includes the specific features of Plotly.js that your Dash app uses. Such bundles are smaller than the full Plotly.js bundles and can therefore improve your app's loading time.
+
+    ## Rendering LaTeX inside `dcc.Graph` Figures
+
+    To use the built-in Plotly.js capability of rendering LaTeX inside figure labels, the `external_script` and `assets` functionality described above can be used together as follows:
+
+        1. add `external_scripts=["https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/MathJax.js?config=TeX-MML-AM_CHTML" ]` to the `app = dash.Dash()` call
```
Comment (archmoj) [inline dash_docs/chapters/external_resources/index.py:376]: Perhaps safer to use mathjax at `2.7.5` as it was used in plotly.py?

Comment (nicolaskruchten) [reply dash_docs/chapters/external_resources/index.py]: done.

**Comment by nicolaskruchten** [comment]
@alexcjohnson i think the crypto thing just got fixed in master and I rebased.

File: dash_docs/chapters/dash_core_components/Graph/index.py [hunk 2]
```diff
@@ -108,6 +108,33 @@
...
+
+    ## Controlling the Plotly.js Version Used by `dcc.Graph`
+
+    The `Graph` component leverages the [Plotly.js](https://plotly.com/javascript/) library to render
+    visualizations. The `Graph` component comes with its own version of the  Plotly.js library, but this can be overridden by placing a Plotly.js bundle
+    in the `assets` directory as described above.
+
+    This technique can be used to:
+
+        * take advantage of new features in a version of Plotly.js that is **more recent** than the one that is included in the currently-installed version of `dash_core_components` (or Dash Enterprise Design Kit).
```
Comment (alexcjohnson) [inline dash_docs/chapters/dash_core_components/Graph/index.py:123]: These lists are getting interpreted as code - looks like they need to be indented the same as the line above.
Screen Shot 2021-02-10 at 1 33 09 PM

**Comment by nicolaskruchten** [comment]
@alexcjohnson ready to go!

**Comment by nicolaskruchten** [comment]
Question: how do I make internal links that also work with DE docs?

File: dash_docs/chapters/dash_core_components/Graph/index.py [hunk 3]
```diff
@@ -108,6 +108,34 @@
...
+    * use [a Plotly-distributed Plotly.js partial bundle](https://github.com/plotly/plotly.js/blob/master/dist/README.md) or [a custom-built Plotly.js bundle](https://github.com/plotly/plotly.js/blob/master/BUILDING.md) which only includes the subset of Plotly.js features that your Dash app uses. Such bundles are smaller than the full Plotly.js bundles that come with the `Graph` component and can therefore improve your app's loading time.
+
+    ## Rendering LaTeX inside `dcc.Graph` Figures
+
+    To use the built-in Plotly.js capability of rendering LaTeX inside figure labels, the [`external_script`
+    functionality can be used](/external-resources): add `external_scripts=["https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML" ]` to the `app = dash.Dash()` call.
+
+    If loading files from a CDN is undesirable, MathJax will have to be installed into the `assets` directory and [the HTML Index Template](/external-resources) will need to be customized
```
Comment (alexcjohnson) [inline dash_docs/chapters/dash_core_components/Graph/index.py:132]: Have you tested putting MathJax into `assets`? I bet it also needs to be included in `assets_ignore` so we don't try to recursively include everything there, let it orchestrate that itself.

**Review by alexcjohnson — APPROVED** [review]
💃 once the note about putting MathJax in `assets` is removed (at some point if necessary we can test this, including `assets_ignore`, and add it back in)

**Pull request merged**